watertap/data/techno_economic/coag_and_floc.yaml

```diff
@@ -7,8 +7,47 @@ default:
   default_removal_frac_mass_solute:
     value: 0
     units: dimensionless
-  removal_frac_mass_solute:
-    tss:
-      value: 0.95
-      units: dimensionless
-      constituent_longform: Total Suspended Solids (TSS)
+  alum_dose:
+    value: 10
+    units: mg/L
+    constituent_longform: Aluminum Sulphate Dosage
+  polymer_dose:
+    value: 0.1
+    units: mg/L
+    constituent_longform: Polymer Dosage
+  anion_to_cation_polymer_ratio:
+    value: 1
+    units: dimensionless
+  rapid_mix_retention_time:
+    value: 5.5
+    units: s
+  floc_retention_time:
+    value: 12
+    units: min
+  num_rapid_mixers:
+    value: 1
+    units: dimensionless
+  num_floc_mixers:
+    value: 3
+    units: dimensionless
+  num_rapid_mix_processes:
+    value: 1
+    units: dimensionless
+  num_floc_processes:
+    value: 2
+    units: dimensionless
+  num_coag_processes:
+    value: 1
+    units: dimensionless
+  num_floc_injection_processes:
+    value: 1
+    units: dimensionless
+  G_rapid_mix:
```

Comment:
I would suggest more descriptive names here.

Reply:
I also consider "velocity_gradient_rapid_mix" but "G" is well known in the discipline when it comes to mixing energy calculations, so I opted for that. Would you prefer I change it to the former?

Reply:
The IDAES standard has always been to be verbose for clarity. It also helps make things more accessible to those new to the domain,